src/XliffTasks.Tests/XamlRuleDocumentTests.cs

```diff
@@ -10,6 +10,524 @@ namespace XliffTasks.Tests
+                  Description="The version of the language available to code in this project."
+                  HelpUrl="https://aka.ms/csharp-versions"
+                  ReadOnly="true"
+                  Category="Advanced">
+    <StringProperty.ValueEditors>
+      <ValueEditor EditorType="String">
+        <ValueEditor.Metadata>
+          <NameValuePair Name="ShowEvaluatedPreviewOnly" Value="True" />
+        </ValueEditor.Metadata>
+      </ValueEditor>
+    </StringProperty.ValueEditors>
+  </StringProperty>
+
+  <BoolProperty Name="CheckForOverflowUnderflow"
+                DisplayName="Check for arithmetic overflow"
+                Description="Throw exceptions when integer arithmetic produces out of range values."
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2166113"
+                Category="Advanced">
+    <BoolProperty.Metadata>
+      <NameValuePair Name="SearchTerms" Value="checked;unchecked" />
+    </BoolProperty.Metadata>
+  </BoolProperty>
+
+  <BoolProperty Name="Deterministic"
+                DisplayName="Deterministic"
+                Description="Produce identical compilation output for identical inputs."
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2165770"
+                Category="Advanced" />
+
+  <EnumProperty Name="ErrorReport"
+                DisplayName="Internal compiler error reporting"
+                Description="Send internal compiler error (ICE) reports to Microsoft."
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2165771"
+                Category="Advanced">
+    <EnumProperty.DataSource>
+      <DataSource HasConfigurationCondition="False" />
+    </EnumProperty.DataSource>
+    <EnumValue Name="none"
+               DisplayName="Never send" />
+    <EnumValue Name="prompt"
+               DisplayName="Prompt before sending" />
+    <EnumValue Name="queue"
+               DisplayName="Queue" />
+    <EnumValue Name="send"
+               DisplayName="Send" />
+  </EnumProperty>
+
+  <EnumProperty Name="FileAlignment"
+                DisplayName="File alignment"
+                Description="Specifies, in bytes, where to align the sections of the output file."
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2166114"
+                Category="Advanced">
+    <EnumValue Name="512"
+               DisplayName="512" />
+    <EnumValue Name="1024"
+               DisplayName="1024" />
+    <EnumValue Name="2048"
+               DisplayName="2048" />
+    <EnumValue Name="4096"
+               DisplayName="4096" />
+    <EnumValue Name="8192"
+               DisplayName="8192" />
+  </EnumProperty>
+
+</Rule>
+""";
+        
+        XamlRuleDocument document = new();
+        StringWriter writer = new();
+        document.Load(new StringReader(source));
+
+        XlfDocument xliffDocument = new();
+        xliffDocument.LoadNew("fr");
+        xliffDocument.Update(document, "test.xaml");
```

Comment:
What is this test covering? The input is so huge it's hard to see what task it's performing, and there doesn't seem to be any expected output.